precompiles/bank/method_balanceof.go

```diff
@@ -0,0 +1,49 @@
+package bank
```

Comment:
Shouldn't the name be `method_balance_of.go`?

Reply:
You're right, in "pure" Go that's the name, renaming it.